Preloader.js

```diff
@@ -1,43 +0,0 @@
-
```

Comment:
Preloader was moved into Game.js as a function instead of this weird stand-alone object